kuma/banners/models.py

```diff
@@ -20,7 +20,8 @@ class Banner(models.Model):
     title = models.CharField('Banner Title', max_length=100)
     main_copy = models.TextField('Main Copy', max_length=200)
     button_copy = models.CharField('Button Copy', max_length=50)
-    theme = models.CharField('Theme', max_length=20,
+    button_url = models.URLField('URL')
+    theme = models.CharField('Theme', max_length=40,
                              choices=THEMES, default=THEME_DEFAULT)
```

Comment:
Be very mindful of using `choices` in model fields. Any edits of any of these choices will require a new DB migration. It's not expensive in terms of processing to execute these migrations but it will cause more noise and more files and potentially confusing merge conflicts if there are more migration files.

Reply:
What I mean by that is; if the list is really very unlikely to change keep it in the model as class attributes. If it's possible that it will change once or thrice a year, consider moving it to some admin.py file or something.

Reply:
Is there another `Field` type on can use that will use `select` as its widget? From what I understand, adding `choices` to a `CharField` is generally how this is accomplished.

Reply:
It is indeed but there are other ways to set it too. For example in a form class. 
I don't think you need to change anything at this point. Just something to be mindful about if you can foresee that there *will* be a lot of flux on this `choices`.